package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@phillips/seldon",
-  "version": "1.9.0",
+  "version": "1.9.2",
```

Comment:
Weird that it updated to 1.9.2 in your branch. This is at 1.9.3 in main. We may have to try to check this out from main and push up to get rid of this file in your changeset